syntax = "proto3";

package blueberry.facade.neighbor;

import "google/protobuf/empty.proto";

service NeighborFacade {
  rpc EnablePageScan(EnableMsg) returns (google.protobuf.Empty) {}
}

message EnableMsg {
  bool enabled = 1;
}
